modules/wpml/wpml-config.php

```diff
@@ -225,7 +236,10 @@ public function translate_types( $types, $hide ) {
 			if ( is_array( $pts ) ) {
 				foreach ( $pts as $pt ) {
 					$attributes = $pt->attributes();
-					if ( 1 == $attributes['translate'] && ! $hide ) {
+					if ( empty( $attributes ) ) {
+						continue;
+					}
+					if ( '1' === (string) $attributes['translate'] && ! $hide ) {
```

Comment:
Why do we cast it ? What type is this initially ?

Reply:
`SimpleXMLElement` https://www.php.net/manual/fr/simplexmlelement.attributes.php 
So we need to cast for the [SimpleXMLElement::__toString()](https://www.php.net/manual/fr/simplexmlelement.tostring.php) would be aplied to read the attribute value.

For other PHPStan reason we have to pass the lazy '==' comparison to a strict comparison therefore casting this required.

BTW, I see other lazy comparison elsewhere in this class.

Reply:
Does it work if we cas to `int`? It would look more logical to me.

Reply:
Done in e525cf49b3de1c932b6c11ab3e943ca929402f6c